File: Pyramid/Pyramid.java
java.util.Scanner;

public class Pyramid {

  public static void main(String[] args) {
    int height;
    Scanner input =  new Scanner(System.in);

    // Prompt user for height of pyramid
    System.out.print("How tall do you want to pyramid to be? ");
    height = input.nextInt();

    // Loop to make each row of pyramid
    for (int i = 0; i < height; i++){
      // Nested loop for the spacing on each row
      for (int x = height; x > i+1; x--) {
        System.out.print(" ");
      }
      // Nested loop to make the '*' on each row
      for (int y = 0; y <= i+i; y++) {
        System.out.print("*");
      }
      // New line at the end of each row
      System.out.print("\n");
    }

  }
  
}

File: Pyramid/Pyramid.py
def main():
    
    height = int(input("How tall do you want the pyramid to be? "))

    # Loop for each row of pyramid
    for i in range(height):

        # Loop for the spacing of each row
        for x in range(i, height):
            print(" ",end="")

        # Loop to print the '*'s on each row
        for y in range((i*2)+1):
            print("*",end="")
        # New line at the end of each row
        print("")

        
if __name__ == "__main__":
    main()
